### PR TITLE
Convert logstash_config variables to attributes

### PR DESCRIPTION
**Pull request by phoolish**
- make template_cookbook an attribute
- make template_variables an attribute

### CODE REVIEW

File: recipes/logstash.rb [hunk 1]
```diff
@@ -50,15 +38,15 @@
 unless node.run_state['lumberjack_decoded_certificate'].nil? || node.run_state['lumberjack_decoded_certificate'].nil?\
  || node['elkstack']['config']['agent_protocol'] != 'lumberjack'
   node.default['elkstack']['config']['logstash']['server']['my_templates']['input_lumberjack'] = 'logstash/input_lumberjack.conf.erb'
-  template_variables['input_lumberjack_ssl_certificate'] = node['lumberjack']['ssl_cert_path']
-  template_variables['input_lumberjack_ssl_key'] = node['lumberjack']['ssl_key_path']
+  node.default['elkstack']['config']['logstash']['server']['my_template_variables']['input_lumberjack_ssl_certificate'] = node['lumberjack']['ssl_cert_path']
+  node.default['elkstack']['config']['logstash']['server']['my_template_variables']['input_lumberjack_ssl_key'] = node['lumberjack']['ssl_key_path']
```
Comment (martinb3) [inline recipes/logstash.rb]: I think these should be `node`, not `node.default` here.

Comment (phoolish) [inline recipes/logstash.rb]: I thought we had to set a precedence level when defining an attribute in a recipe.

Comment (martinb3) [inline recipes/logstash.rb]: Ah, you're right. Didn't scroll to the right :(

Comment (phoolish) [inline recipes/logstash.rb]: been there done that.

**Comment by martinb3** [comment]
Thanks @phoolish!

**Pull request merged**